units.js

```diff
@@ -100,4 +100,12 @@ module.exports = {
     base: 0.219969,
     aliases: ['gallon', 'gallons'],
   },
+  kmpl: {
+    base: 1,
+    aliases: ['km/l'],
+  },
+  mpg: {
```

Comment:
should we note that this is us gallons and not British gallons :roll_eyes: 

https://www.calculateme.com/gas-mileage/us-mpg-to-km-per-liter

Reply:
fixed in read me.